src/privileges/helpers.js

```diff
@@ -153,18 +153,18 @@ function moveToFront(groupNames, groupToMove) {
 	}
 }
 
-helpers.giveOrRescind = async function (method, privileges, cids, groupNames) {
-	groupNames = Array.isArray(groupNames) ? groupNames : [groupNames];
+helpers.giveOrRescind = async function (method, privileges, cids, members) {
+	members = Array.isArray(members) ? members : [members];
 	cids = Array.isArray(cids) ? cids : [cids];
-	for (const groupName of groupNames) {
+	for (const member of members) {
 		const groupKeys = [];
 		cids.forEach((cid) => {
 			privileges.forEach((privilege) => {
-				groupKeys.push('cid:' + cid + ':privileges:groups:' + privilege);
```

Comment:
`groups:` prefix removed here so this function works for both groups and uids
To give find privilege to a uid 1
`privileges.categories.give(['find'], cid, 1);`
To give groups:find privilege to a group
`privileges.categories.give(['groups:find'], cid, 'my group');`